ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -1364,9 +1365,14 @@ private void assertBlobReadSuccess(GetBlobOptions options, Future<Long> readInto
       } else {
         putContentBuf = ByteBuffer.wrap(putContent);
         // If a range is set, compare the result against the specified byte range.
-        if (options != null && options.getRange() != null) {
-          ByteRange range = options.getRange().toResolvedByteRange(blobSize);
-          putContentBuf = ByteBuffer.wrap(putContent, (int) range.getStartOffset(), (int) range.getRangeSize());
+        if (options != null) {
+          if (options.getBlobSegment() != NO_BLOB_SEGMENT_IDX_SPECIFIED) {
+
```

Comment:
Are you going to do something here?

Reply:
I was but the new code gets tested in GetManagerTest instead